File: Programming EF Code First/chapter5/Chapter5 CSharp/BAGA_FluentAPI/DataAccess/ActivityConfiguration.cs
using System.Data.Entity.ModelConfiguration;
using Model;
using System.ComponentModel.DataAnnotations;

namespace DataAccess
{
  public class ActivityConfiguration : EntityTypeConfiguration<Activity>
  {
    public ActivityConfiguration()
    {
      Property(a => a.Name)
        .IsRequired()
        .HasMaxLength(50);
    }
  }
}

File: Programming EF Code First/chapter6/Chapter6/Chapter 6 VB/Fluent/DataAccess/ActivityConfiguration.vb
Imports System.Data.Entity.ModelConfiguration
Imports Model
Imports System.ComponentModel.DataAnnotations

Namespace DataAccess
  Public Class ActivityConfiguration
	  Inherits EntityTypeConfiguration(Of Activity)
	Public Sub New()
      Me.Property(Function(a) a.Name).IsRequired().HasMaxLength(50)
	End Sub
  End Class
End Namespace